EnglishTranslation-master/ITEM.tsv

```diff
@@ -5281,7 +5281,7 @@ ITEM_20150714_005280	Belfigi Bead
 ITEM_20150714_005281	There is a little color difference between Dimmer beads. {nl}Obtained from Belfigi.
 ITEM_20150714_005282	Cliquare Core
 ITEM_20150714_005283	A mutant variation of the Nucl, the power source of mutant monster. Contains highly condensed magic that might explode if handled carelessly. {nl}Obtain from Stone Orcas.
-ITEM_20150714_005284	A record written by Hones and Liars's teacher. It seems to contain very important information. 
+ITEM_20150714_005284	A record written by Hones and Liars teacher. It seems to contain very important information. 
```

Comment:
A record written by the teacher of Hones and Raius. It seems to contain very important information.